halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/metricStores/prometheus/PrometheusStore.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.halyard.config.model.v1.metricStores.prometheus;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.netflix.spinnaker.halyard.config.model.v1.node.MetricStore;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+public class PrometheusStore extends MetricStore {
+  @Override
+  public String getNodeName() {
+    return "prometheus";
+  }
+
+  @JsonProperty("push_gateway")
+  private String pushGateway;
```

Comment:
Since this exists only here & won't require any other changes to remove I'm going to merge this now. Both dpeach & jtk54 are blocked on this PR being merged.